data/out/tables/sample1.csv.manifest

```diff
@@ -0,0 +1 @@
+{"destination":"out.c-main.Leads","columns":["id",["id","auto_col_1","auto_col_2","auto_col_3"]],"incremental":false,"primary_key":[],"delimiter":",","enclosure":"\"","metadata":[{"id":"228956","key":"KBC.createdBy.component.id","value":"keboola.python-transformation","provider":"system","timestamp":"2017-05-26 00:39:07"}],"column_metadata":{"id":[]}}
```

Comment:
chybí newline

Reply:
kde?

Reply:
Na konci toho souboru.

Reply:
na co? jak to JsonEncoder produkuje, tak to ukladam

Reply:
Je to best practice aby to neudělalo tohle:
```
root@zion $ cat some.json
{"lorem":"ipsum"}root@zion $
```

https://github.com/keboola/php-component/blob/master/src/Manifest/ManifestManager.php#L182

Reply:
no ja nevim, je to interni soubor, nikdo si ho nema co catovat, kdyz tam dam newline tak to na me dela dojem, ze tam snad jako musi byt aby to fungovalo a ne ze tam jen jen tak, kdyby nahodou nekdo hackoval server a catnul si manifest

Reply:
ono to není jen tohle... třeba i `wc -l some.json` bude 0 a prostě jsou s tím obecně zmatky. Ale tady přímo je to jedno, to souhlas. Jen, že bude v diffu `\ No newline at the end of the file`